tests/oauth2/OAuth2SummitEventsApiTest.php

```
@@ -477,6 +480,67 @@ public function testUpdateEvent()
+        $content = $response->getContent();
+        $event = json_decode($content);
+        $this->assertEquals('Updated Draft Title', $event->title);
+        $this->assertNotEquals(Presentation::PHASE_COMPLETE, $event->progress);
+        $this->assertNotEquals(Presentation::STATUS_RECEIVED, $event->status);
+    }
+
+    public function testUpdateDraftEventOnPublishedPresentationReturns412()
+    {
+        $presentation = self::$presentations[0];
+
+        $response = $this->action(
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$response'. (undefined)

(UnusedLocalVariable)